reverse_linkedlist/jade_solution.py

```diff
@@ -0,0 +1,41 @@
+def get_data_list(data):
+  data_list = []
+  data_list.append(one['data'])
+  inner_dict = one['next']
```

Comment:
What do you think about naming this `current_node`?  I'm not sure how to interpret `inner_dict` when I read it.

Reply:
no!